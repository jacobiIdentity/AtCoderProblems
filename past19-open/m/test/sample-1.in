4 4
2 5 3 1
1 3 0.500000
1 2 0.250000
2 3 0.250000
3 4 0.500000
