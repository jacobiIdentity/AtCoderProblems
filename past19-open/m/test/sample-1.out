3.7500000000
